.travis.yml

```diff
@@ -9,6 +9,7 @@ branches:
   except:
     - piptools-ignore-patch
 install:
+  - "python -m pip install pip --upgrade"
```

Comment:
Isn't this redundant with line 6 of `tests/review.t`?

      $ pip install --upgrade --force-reinstall 'pip' > /dev/null 2>&1